inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/InlongGroupController.java

```diff
@@ -166,4 +167,10 @@ public Response<InlongGroupTopicInfo> getTopic(@PathVariable String groupId) {
         return Response.success(groupService.getTopic(groupId));
     }
 
+    @RequestMapping(value = "reset", method = RequestMethod.POST)
+    @ApiOperation(value = "Reset group status when group is in CONFIG_ING|SUSPENDING|RESTARTING|DELETING")
+    public Response<Boolean> reset(@RequestBody InlongGroupResetRequest request) {
```

Comment:
Add @Validation before @RequestBody, otherwise the validation will not take effect

Reply:
1